wycheproof/src/lib.rs

```diff
@@ -28,20 +28,36 @@ pub mod eddsa {
     }
 
     #[allow(dead_code)]
-    pub struct EddsaTestGroup<'a> {
-        pub kind: &'a str,
-        pub key: &'a Key<'a>,
-        pub tests: &'a [&'a SignatureTestVector<'a>],
+    pub struct XdhTestVector<'a> {
+        pub tc_id: u32,
```

Comment:
`test_case_id`?

Reply:
In this particular case I didn't change the name for now, as it matches the Wycheproof JSON.